src/thor/costmatrix.cc

```diff
@@ -328,7 +328,7 @@ void CostMatrix::ForwardSearch(const uint32_t index, const uint32_t n, GraphRead
       *es = {EdgeSet::kTemporary, idx};
       edgelabels.emplace_back(pred_idx, edgeid, oppedge, directededge, newcost, mode_, tc,
                               pred.path_distance() + directededge->length(),
-                              (pred.not_thru_pruning() || !directededge->not_thru()),
+                              (pred.not_thru_pruning() || !directededge->not_thru()), true,
```

Comment:
why hardcode this? i think this algorithm should work the same as bidirectional_astar, indeed you can see the not_thru stuff which is an analog in the other arguments to the edgelabel constructor

Reply:
I'll fix this (oversight on my end).